packages/flutter/lib/src/gestures/binding.dart

```diff
@@ -19,6 +19,7 @@ import 'pointer_router.dart';
 import 'pointer_signal_resolver.dart';
 import 'resampler.dart';
 
+/// Sample time change callback.
```

Comment:
normally we mention where this would get used in the doc.

Reply:
+1, see also https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#leave-breadcrumbs-in-the-comments:

> Typedefs should mention at least one place where the signature is used.

Reply:
It looks like this is only used internally.  Maybe it should be made private?

Reply:
I was just about to say that. Looks like this should actually be private.

Reply:
Should I go ahead and make it private here or is that better in a different PR?

Reply:
Just do it here :)

Reply:
Privatized.